bears/r/FormatRBear.py

```diff
@@ -31,10 +31,10 @@ class FormatRBear:
     @staticmethod
     @deprecate_settings(indent_size='tab_width')
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
-                         r_keep_blank_lines: bool=True,
-                         r_braces_on_next_line: bool=False,
-                         r_use_arrows: bool=False,
+                         r_keep_comments: bool=None,
+                         r_keep_blank_lines: bool=None,
+                         r_braces_on_next_line: bool=None,
+                         r_use_arrows: bool=None,
```

Comment:
It was right (using `None`). The original issue of this PR is making FormatRBear _ignore_ those option. Meaning they dont enforce using arrow (`True`) neither never user arrow (`False`).